plugins/ssh-plugin/src/ssh-plugin-backend.ts

```diff
@@ -237,6 +246,40 @@ const createKeyPair = async (sshkeyManager: SshKeyManager) => {
     }
 };
 
+const uploadPrivateKey = async (sshkeyManager: SshKeyManager) => {
+    let hostName = await getHostName('Please provide a hostname (e.g. github.com) or leave empty to setup default name');
+    if (!hostName) {
+        hostName = `default-${Date.now()}`;
+    }
```

Comment:
I think we need to make this approach as a default for other methods e.g. create key, generate key.

Reply:
Fixed